ister_gui.py

```diff
@@ -2395,10 +2474,18 @@ def _save_config(self, config, mount_d):
                     output = ''
                 if 'TYPE="' in output:
                     config['FilesystemTypes'][-1]['disable_format'] = True
+            if "encryption" in mount_d[point]:
+                config['FilesystemTypes'][-1]["encryption"] = {
+                    "passphrase": mount_d[point]['encryption'],
```

Comment:
Since this is writing the passphrase to disk (and may be reused or given to somebody else) we should make sure to securely delete this file after the installer has finished running.

Reply:
the whole template I suppose

Reply:
I am working in change how ister.py is called in order to keep passphrase hide, I mean there is an import for ister in the top of ister_gui.py then call install_os must be to easy, I will change some thinks in order to make this posible, tha way how ister.py and ister_gui.py are called will be keeped

Reply:
done